common/src/main/java/com/wynntils/utils/render/Texture.java

```diff
@@ -6,180 +6,183 @@
-    CHANGELOG_BACKGROUND("changelog.png", 294, 198),
-    COSMETIC_VIEWER_BACKGROUND("cosmetics_viewer_background.png", 72, 92),
-    EMERALD_COUNT_BACKGROUND("emerald_count_background.png", 24, 24),
-    EXPERIENCE_BAR("bars_exp.png", 256, 256),
-    FAVORITE("favorite.png", 18, 18),
-    GEAR_ICONS("gear_icons.png", 64, 128),
-    GEAR_VIEWER_BACKGROUND("gear_viewer_background.png", 103, 92),
-    HEALTH_BAR("bars_health.png", 256, 256),
-    HEALTH_BAR_OVERFLOW("bars_health_overflow.png", 256, 256),
-    HIGHLIGHT("highlight.png", 256, 256),
-    ITEM_LOCK("item_lock.png", 16, 16),
-    LOOTRUN_LINE("path_arrow.png", 16, 16),
-    MANA_BAR("bars_mana.png", 256, 256),
-    MANA_BAR_OVERFLOW("bars_mana_overflow.png", 256, 256),
-    OVERLAY_SELECTION_GUI("overlay_selection_gui.png", 195, 256),
-    UNIVERSAL_BAR("universal_bar.png", 81, 16),
-    VIGNETTE("vignette.png", 512, 512),
-    WYNNCRAFT_ICON("wynncraft_icon.png", 64, 64),
-    PLAYER_INFO_OVERLAY("tab_overlay.png", 385, 229),
-
-    // Discovery Screen
-    DISCOVERED_SECRET("discovered_secret.png", 19, 36),
-    DISCOVERED_SECRET_ICON("discovered_secret_icon.png", 5, 7),
-    DISCOVERED_TERRITORY("discovered_territory.png", 17, 40),
-    DISCOVERED_TERRITORY_ICON("discovered_territory_icon.png", 8, 7),
-    DISCOVERED_WORLD("discovered_world.png", 18, 34),
-    DISCOVERED_WORLD_ICON("discovered_world_icon.png", 5, 7),
-    UNDISCOVERED_SECRET("undiscovered_secret.png", 17, 32),
-    UNDISCOVERED_SECRET_ICON("undiscovered_secret_icon.png", 11, 7),
-    UNDISCOVERED_TERRITORY("undiscovered_territory.png", 21, 38),
-    UNDISCOVERED_TERRITORY_ICON("undiscovered_territory_icon.png", 6, 7),
-    UNDISCOVERED_WORLD("undiscovered_world.png", 21, 34),
-    UNDISCOVERED_WORLD_ICON("undiscovered_world_icon.png", 11, 7),
-
-    // Settings Screen
-    SETTING_BACKGROUND("book.png", 400, 220),
-    SETTING_SCROLL_AREA("book_scrollarea.png", 168, 175),
-    SETTING_SCROLL_BUTTON("book_scroll_button.png", 5, 18),
-
-    // Quest book related
-    BACK_ARROW("back_arrow.png", 32, 9),
-    BACKWARD_ARROW("backward_arrow.png", 36, 10),
-    DIALOGUE_BUTTON("dialogue_button.png", 14, 11),
-    FORWARD_ARROW("forward_arrow.png", 36, 10),
-    QUEST_BOOK_BACKGROUND("quest_book.png", 339, 220),
-    QUESTS_BUTTON("quests_button.png", 16, 16),
-    QUEST_BOOK_SEARCH("quest_book_search.png", 133, 23),
-    QUEST_BOOK_TITLE("quest_book_title.png", 168, 33),
-    RELOAD_BUTTON("reload_button.png", 40, 20),
-    SORT_ALPHABETICALLY("sort_alphabetical.png", 23, 46),
-    SORT_DISTANCE("sort_distance.png", 23, 46),
-    SORT_LEVEL("sort_level.png", 23, 46),
-
-    // Quest State Icons
-    QUEST_CANNOT_START("quest_cannot_start_icon.png", 7, 7),
-    QUEST_CAN_START("quest_can_start_icon.png", 11, 7),
-    QUEST_FINISHED("quest_finished_icon.png", 11, 7),
-    QUEST_STARTED("quest_started_icon.png", 7, 7),
-
-    // Caves screen
-    CAVE_AVALIABLE("cave_avaliable.png", 7, 7),
+    // Character Selection
```

Comment:
Just a small nit? Can you use `// region ` and `// endregion`? Just so it's easier to read this.

Reply:
I've also added a new line above the "sub regions" to make that a bit easier to read, or should they be defined as seperate regions?

Reply:
I think this is fine. Maybe the icon one is a bit too broad, but otoh having too small regions is also annoying when trying to use the IntelliJ structure view.